.travis.yml

```diff
@@ -1,14 +1,23 @@
 sudo: required
+dist: xenial
 language: go
 go:
 - "1.10"
+#- "1.12.x"
+go_import_path: github.com/openshift/cluster-monitoring/operator
+#env:
+#  - GO111MODULE=on
+#cache:
```

Comment:
Are you trying something out here, or is this intentionally commented out?

Reply:
trying out ;)